mobile/lib/modules/search/services/person.service.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:immich_mobile/shared/providers/api.provider.dart';
+import 'package:immich_mobile/shared/services/api.service.dart';
+import 'package:openapi/api.dart';
+
+final personServiceProvider = Provider(
+  (ref) => PersonService(
+    ref.watch(apiServiceProvider),
+  ),
+);
+
+class PersonService {
+  final ApiService _apiService;
+
+  PersonService(this._apiService);
+
+  Future<List<PersonResponseDto>?> getCuratedPeople() async {
+    try {
+      return await _apiService.personApi.getAllPeople();
+    } catch (e) {
+      debugPrint("Error [getCuratedPeople] ${e.toString()}");
+      return null;
+    }
+  }
+
+  Future<List<AssetResponseDto>?> getPersonAssets(String id) async {
```

Comment:
The service should already do the napping from DTO to asset. So provider and widgets never use Dtos and only our own classes